Kernel/Devices/HID/HIDManagement.cpp

```diff
@@ -110,7 +110,8 @@ UNMAP_AFTER_INIT void HIDManagement::enumerate()
     if (ACPI::Parser::the() && !ACPI::Parser::the()->have_8042())
```

Comment:
commit message nitpicks:
`Not only it makes the code` => `Not only does it makes the code`
`it allows to enforce` => `it allows us to enforce`